### PR TITLE
Public api docs for All my sons api

### PR DESCRIPTION
**Pull request by rajeev2626**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+    + BusinessId (string, required) ... the Business ID
+    + api_key (string, required) ... the API key
+
+### Get Reviews Count By Driver [POST]
+
+#### Request
+
+<center><b>Example curl request</b></center>
+
+    curl -XPOST -H "Content-type: application/json" -d '{"businessNumbers": [145308764185002,145308769721320],"sourceAlias": ["google","citysearch"],"employees": [{"firstName": "String","lastName": "String","phone": "String"}],"fromDate": "04/01/2013","toDate": "05/3/2015"}' 'https://api.birdeye.com/resources/v1/review/report/count-by-rating/employee/145308769721320?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451'
```
Comment (shubham-agarwal15) [inline apiary.apib:6404]: Should be like curl -H "content-type: application/json" -H "accept:application/json" -X POST to keep uniformity with existing documentation.

File: apiary.apib [hunk 2]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+    + BusinessId (string, required) ... the Business ID
+    + api_key (string, required) ... the API key
+
+### Get Reviews BY Count [POST]
+
+#### Request
+
+<center><b>Example curl request</b></center>
+
+    curl -XPOST -H "Content-type: application/json" -d '{"businessNumbers": [145308764185002,145308769721320],"sourceAlias": ["google","citysearch"],"fromDate": "04/01/2013","toDate": "05/3/2015"}' 'https://api.birdeye.com/resources/v1/review/report/count-by-rating/145308769721320?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451'
```
Comment (shubham-agarwal15) [inline apiary.apib:6216]: Should be like curl -H "content-type: application/json" -H "accept:application/json" -X POST to keep uniformity with existing documentation.

File: apiary.apib [hunk 3]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+       145308769721320
+      ],
+     "sourceAlias": [
+     "google","citysearch"
+     ],
+     "employees": [
+       {
+      "firstName": "String",
+      "lastName": "String",
+      "phone": "String"
```
Comment (shubham-agarwal15) [inline apiary.apib:6420]: Declare a valid request body as this is an _Example payload_, do not use "String"

File: apiary.apib [hunk 4]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+        </tr>
+        <tr>
+            <td>toDate</td>
+            <td>Review date (end) e.g. "04/01/2013"</td>
+        </tr>
+        <tr>
+            <td>sourceAlias</td>
+            <td>Aggregation source name e.g. "google","citysearch" etc. Note: use "our_website" to filter reviews posted on Birdeye platform or use "direct_feedback" for direct feedback. </td>
+        </tr>
+         <tr>
```
Comment (shubham-agarwal15) [inline apiary.apib:6449]: As mentioned for above API, describe the optional fields

File: apiary.apib [hunk 5]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+            <td>countByRating</td>
+            <td>Review's count against the rating</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+For facebook sourcetype, following table can be used to check recommendations.
```
Comment (shubham-agarwal15) [inline apiary.apib:6526]: **sourceAlias** should be used instead of sourceType for uniform usage of keywords

**Review by shubham-agarwal15 — CHANGES_REQUESTED** [review]

File: apiary.apib [hunk 6]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
 + Response 400
 
     [2078][]
+    
+
+## Get [/v1/review/report/count-by-rating/{BusinessId}{?api_key}]
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib [hunk 7]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+Business review(s) from BirdEye platform can be fetched using following filter in request payload:
+
++ Parameters
+
+    + BusinessId (string, required) ... the Business ID
+    + api_key (string, required) ... the API key
+
+### Get Reviews BY Count [POST]
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib [hunk 8]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+            <td>fromDate</td>
+            <td>Review date (start) e.g. "04/01/2013"</td>
+        </tr>
+        <tr>
+            <td>toDate</td>
+            <td>Review date (end) e.g. "04/01/2013"</td>
+        </tr>
+        <tr>
+            <td>sourceAlias</td>
+            <td>Aggregation source name e.g. "google","citysearch" etc. Note: use "our_website" to filter reviews posted on Birdeye platform or use "direct_feedback" for direct feedback. </td>
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib [hunk 9]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+    {
+      "rating": 2,
+      "count": Integer
+    },
+    {
+      "rating": 3,
+      "count": Integer
+    },
+    {
+      "rating": 4,
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib [hunk 10]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>reviewCount</td>
+            <td>Total number reviews</td>
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib [hunk 11]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+        </tr>
+         <tr>
+            <td>countByRating</td>
+            <td>Review's count against the rating</td>
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib [hunk 12]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+For facebook sourcetype, following table can be used to check recommendations.
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib [hunk 13]
```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
...
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1228</td>
+            <td>Invalid updated to date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+## Get [/v1/review/report/count-by-rating/employee/{BusinessId}{?api_key}]
```
[the remark this answers is not in the document]
Comment (rajeev2626) [reply apiary.apib]: Done .. We decided to keep it under reviews tab

File: apiary.apib (hunk 1 above)
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib (hunk 2 above)
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib (hunk 3 above)
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib (hunk 4 above)
Comment (rajeev2626) [reply apiary.apib]: Done

File: apiary.apib (hunk 5 above)
Comment (rajeev2626) [reply apiary.apib]: Done

**Review by shubham-agarwal15 — APPROVED** [review]

**Pull request merged**